core-bundle/src/ServiceAnnotation/Page.php

```diff
@@ -22,32 +20,53 @@
  *
  * @Annotation
  * @Target({"CLASS", "METHOD"})
- * @Attributes({
- *     @Attribute("value", type = "string"),
- *     @Attribute("path", type = "string"),
- *     @Attribute("urlSuffix", type = "string"),
- *     @Attribute("requirements", type = "array"),
- *     @Attribute("options", type = "array"),
- *     @Attribute("defaults", type = "array"),
- *     @Attribute("methods", type = "array"),
- *     @Attribute("contentComposition", type = "boolean"),
- * })
+ *
+ * @see \Symfony\Component\Routing\Annotation\Route
  */
 final class Page implements ServiceAnnotationInterface
 {
-    /**
-     * @var array
-     */
-    private $attributes;
+    private $type;
```

Comment:
The class properties should have `@var` annotations.

Reply:
(We could also abolish this rule in the future, but for now it still applies.)

Reply:
See #1974.

Reply:
added in 7cc0debe1e3549a1743a89fb9a87475285d769b4